src/aosm/azext_aosm/deploy/deploy_with_arm.py

```diff
@@ -594,14 +596,18 @@ def validate_and_deploy_arm_template(
         if validation_res.error:
             # Validation failed so don't even try to deploy
             logger.error(
-                "Template for resource group %s has failed validation. The message was: %s.\
-                See logs for additional details.",
+                (
+                    "Template for resource group %s has failed validation. The message"
+                    " was: %s.                See logs for additional details."
+                ),
                 resource_group,
                 validation_res.error.message,
             )
             logger.debug(
-                "Template for resource group %s failed validation. \
-                Full error details: %s",
+                (
+                    "Template for resource group %s failed validation.                "
```

Comment:
Again, why so many spaces?

Reply:
Sorry, and thanks for catching. I think that was the merge from main messing things up :-(